unison-src/transcripts/fix2004.output.md

```diff
@@ -98,16 +98,10 @@ Now we fork `a2` off of `a`. `a` continues on, deleting the terms it added previ
 
 .> merge newbranchA a
 
-  Here's what's changed in a after the merge:
-  
-  Added definitions:
+  ✅
   
-    1. dontDelete : Float -> Float -> Float
```

Comment:
@pchiusano  I figured that to be consistent I would just change the default behaviour of all 'merges' to "replace entire history if the head of the dest is empty". This changes the behaviour of this transcript because it no-longer finds the common LCA.

Curious what your preferred way forward is here, the options I can see are:

1. Always 'replace' history when merging over a branch with a completely empty HEAD, meaning this transcript would be updated to the results on the right.
2. Only update the default for the `pull` command, which means that `pull` and `merge` handle the 'merge' step differently from one another.
3. Keep branch history on all merges, but add a new `reset` command for when folks just want to force their local history to match some other history.